ci/Jenkinsfiles/export.groovy

```diff
@@ -212,64 +212,66 @@ pipeline {
               mv values.yaml values.yaml.tosubst
               envsubst < values.yaml.tosubst > values.yaml
             """
-            // second create target namespace (if doesn't exist) and copy secrets to target namespace
-            script {
-              try {
-                boolean nsExists = sh(returnStatus: true, script: "kubectl get namespace ${PREVIEW_NAMESPACE}") == 0
-                if (nsExists) {
-                  // Previous preview deployment needs to be scaled to 0 to be replaced correctly
-                  sh "kubectl --namespace ${PREVIEW_NAMESPACE} scale deployment export --replicas=0"
-                } else {
-                  sh "kubectl create namespace ${PREVIEW_NAMESPACE}"
-                }
-                sh "kubectl --namespace platform get secret kubernetes-docker-cfg -ojsonpath='{.data.\\.dockerconfigjson}' | base64 --decode > /tmp/config.json"
-                sh """kubectl create secret generic kubernetes-docker-cfg \
-                    --namespace=${PREVIEW_NAMESPACE} \
-                    --from-file=.dockerconfigjson=/tmp/config.json \
-                    --type=kubernetes.io/dockerconfigjson --dry-run -o yaml | kubectl apply -f -"""
-                String previewCommand = "jx step helm install --namespace ${PREVIEW_NAMESPACE} --name ${PREVIEW_NAMESPACE} --verbose ."
+            retry (2) {
```

Comment:
Don't you want to only retry the curl commands? The rest (init part and Helm deployment) shouldn't cause any issue I guess.

Reply:
The curl commands already use a "curl retry" see https://github.com/nuxeo/nuxeo-explorer/blob/master/ci/Jenkinsfiles/export.groovy#L118

Here i would like to try again the whole export waiting for the whole system to be resilient: sometimes the site is not reachable, sometimes save fails, sometimes export fails, etc (and also sometimes a valid problem of course): this is the most un-resilient part so this is mainly to avoid having me do a manual retry for these different use cases, for now.

Reply:
As discussed, i put the retry lower and we will see how it goes